documentation/en/cli-lotus.md

```diff
@@ -2208,10 +2208,10 @@ OPTIONS:
 
 ```
 NAME:
-   lotus log list - List log systems
+   lotus-miner log list - List log systems
 
 USAGE:
-   lotus log list [command options] [arguments...]
+   lotus-miner log list [command options] [arguments...]
```

Comment:
I do not think this is correct? If I understand the intent of the changes/PR in general, it seems like we will still have `lotus auth api-info`, `lotus log list`, `lotus log set-level` and `lotus log alerts` commands, and therefor having these being written as lotus-miner CLIs in the Lotus CLI seems wrong?

Reply:
yeah, you're right, good catch; the annoyance is that the command objects are used in both places and they get a certain amount of internal state built up over use, and we use them twice so this name gets sticky; I'll figure out a fix